packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -49,6 +51,24 @@ const MenuInspectorControls = ( {
 		[ clientId ]
 	);
 
+	const experimentMainContent = () => {
```

Comment:
I extracted this as otherwise the different ternarys become hard to grok.

Reply:
Defining components within components should be avoided. Rather let's extract two components in the same file but outside the definition of the `<MenuInspectorControls> components:

1. `<ExperimentControls />`
2. `<DefaultControls />`

This will fix the ternary problem that motivated the change whilst also making the code easier to read and better follow SRP.

Reply:
Thanks for the review. I added a commit for this.